src/main/java/com/google/sps/servlets/PostDataServlet.java

```diff
@@ -78,6 +78,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     newPost.requestToPost(request);
 
     if (newPost.valid) {
+      Entity collegeEntity = new Entity("College", collegeId);
+      datastore.put(collegeEntity);
+
       Entity newPostEntity = newPost.postToEntity(ENTITY_KIND);
       datastore.put(newPostEntity);
```

Comment:
Why are we adding the same post to two different tables?

Reply:
We're not! `collegeEntity` is completely separate from `newPostEntity`!